youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -44,6 +46,8 @@ def check_version(self):
                 self.basename, self.basename, required_version)
             if self._downloader:
                 self._downloader.report_warning(warning)
```

Comment:
>The warning should be printed in `check_version`,  otherwise it's printed when we check it in `YoutubeDL`.

This is intentional. By this we can tell a user that youtube-dl deviates from the "planned" `bestvideo+bestaudio` strategy and encourage him to update. At least this would be a clue for a user wondering why youtube-dl does not download the best overall quality despite avconv is installed.

There is already `is_outdated_version` so I thought `is_outdated` would be misleading.